### PR TITLE
[HF23-AB] Updated descriptions(tooltips) for kanban board, color picker and tree select components

### LINKED ISSUE #7527: [refactor]: Update descriptions(tooltips) for kanban board, color picker and tree select components
### Summary
Descriptions(tooltips) for components need to be updated.

### How to reproduce the issue?

1. Open the ToolJet App-Builder
2. Hover over the components in the Components library on the right
3. You can see descriptions(tooltips) for each component on hover

### Screenshots 

Table component description(tooltip) example:

table-en

table-widget-config

image

### Any possible solutions?

Replace the current labels in the `widgetConfig.js` and `en.json` files according to the below table.

| Name             | Current Description                  | New Description       |
|:-----------------|:---------------------------------|:----------------------|
| Kanban Board     | Kanban Board                     | Task management board          |
| Color Picker     | Color Picker Palette             | Choose colors from a palette     |
| Tree Select      | Select values from a tree view   | Hierarchical item selector     |

### If the bug is confirmed, would you be willing to submit a PR?
No

### Code of Conduct

- [ ] I agree to follow the [ToolJet Code of Conduct](https://docs.tooljet.com/docs/contributing-guide/code-of-conduct)

To seek assistance or resolve issues, we invite you to join our active community of ToolJet developers on [Slack](https://tooljet.com/slack).

### PR DESCRIPTION
**Pull request by Chetax**
Fixes #7527  

# 👨‍💻 Changes proposed(What did you do ?)
Updated the  descriptions(tooltips) for kanban board, color picker and tree select components.

# ✔️ Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

### CODE REVIEW

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey! Thanks for the PR.
For the component `Kanban`, you've made changes to the object "KanbanBoard".
Additionally, you'll need to make the same changes for the object "Kanban" which is missed. Could you please address these required changes?

image

**Comment by Chetax** [comment]
@tejakummarikuntla  can you please review once again.

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -5111,8 +5111,8 @@ ReactDOM.render(<ConnectedComponent />, document.body);`,
   },
   {
     name: 'Kanban',
-    displayName: 'Kanban',
-    description: 'Kanban',
+    displayName: 'Kanban Board',
```
Comment (tejakummarikuntla) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:5114]: Hey @Chetax , please revert this change; the `displayName` value should not be altered. It should be back to 'Kanban'. You can keep the changes in the `description` value.

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey @Chetax, Thanks for addressing the changes, there's one more change needed, please take a look into the comment.

**Pull request merged**